LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1976,6 +1976,229 @@ TEMPLAR_PISTOL_RANGE[23] = -80
 TEMPLAR_PISTOL_RANGE[24] = -90
 TEMPLAR_PISTOL_RANGE[25] = -100
 
++Bullpup_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSMG_Base"
++Bullpup_Coil_ImagePath="UILibrary_LW_Overhaul.InventoryArt.CoilSMG_Base"
+
++BULLPUP_LASER_BASEDAMAGE=	(Damage=4, Spread=1, PlusOne=75, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
+
+
++BULLPUP_LASER_AIM = 5
++BULLPUP_LASER_CRITCHANCE = 0
++BULLPUP_LASER_ICLIPSIZE = 3
++BULLPUP_LASER_ISOUNDRANGE=30
++BULLPUP_LASER_IENVIRONMENTDAMAGE = 0
++BULLPUP_LASER_UPGRADESLOTS=3;
+
++SKIRMISHER_LASER_RANGE[0] = 30
++SKIRMISHER_LASER_RANGE[1] = 30
++SKIRMISHER_LASER_RANGE[2] = 25
++SKIRMISHER_LASER_RANGE[3] = 20
++SKIRMISHER_LASER_RANGE[4] = 15
++SKIRMISHER_LASER_RANGE[5] = 10
++SKIRMISHER_LASER_RANGE[6] = 5
++SKIRMISHER_LASER_RANGE[7] = 0
++SKIRMISHER_LASER_RANGE[8] = 0
++SKIRMISHER_LASER_RANGE[9] = 0
++SKIRMISHER_LASER_RANGE[10] = 0
++SKIRMISHER_LASER_RANGE[11] = 0
++SKIRMISHER_LASER_RANGE[12] = -4
++SKIRMISHER_LASER_RANGE[13] = -8
++SKIRMISHER_LASER_RANGE[14] = -12
++SKIRMISHER_LASER_RANGE[15] = -16
++SKIRMISHER_LASER_RANGE[16] = -20
++SKIRMISHER_LASER_RANGE[17] = -24
++SKIRMISHER_LASER_RANGE[18] = -30
++SKIRMISHER_LASER_RANGE[19] = -40
++SKIRMISHER_LASER_RANGE[20] = -50
++SKIRMISHER_LASER_RANGE[21] = -60
++SKIRMISHER_LASER_RANGE[22] = -70
++SKIRMISHER_LASER_RANGE[23] = -80
++SKIRMISHER_LASER_RANGE[24] = -90
++SKIRMISHER_LASER_RANGE[25] = -100
+
++BULLPUP_COIL_BASEDAMAGE=	(Damage=7,	Spread=2, PlusOne=50,	Crit=4, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
```

Comment:
Would be nice to have this with the other weapon damage entries. Also, would like the crit to be 3 rather than 4. Crit levels should definitely be below assault rifle levels.